src/main/java/org/jenkinsci/plugins/runselector/selectors/SpecificRunSelector.java

```diff
@@ -95,8 +95,13 @@ public String getBuildNumber() {
         return run;
     }
 
-    @Extension(ordinal=-10)
-    public static final Descriptor<RunSelector> DESCRIPTOR =
-            new SimpleRunSelectorDescriptor(
-                SpecificRunSelector.class, org.jenkinsci.plugins.runselector.Messages._SpecificRunSelector_DisplayName());
+    @Symbol("specificRun")
```

Comment:
(Missed in the last review)
Why isn't this "specific" ? Other selectors are defined with names removing "RunSelector" suffixes from their class names.